src/deps-index.ts

```diff
@@ -47,12 +50,14 @@ async function _getTransitiveDependencies(
           }
           throw err;
         }
+        const resolvedHtmlImportUrl = urlLib.resolve(
+            documentBase, urlUtils.relativeUrl(documentBase, htmlImport.url));
```

Comment:
why do we need both `resolve` and `relativeUrl` calls, with `documentBase` appearing twice? Can't `resolve` alone do it?